src/Symfony/Component/Lock/LockFactory.php

```diff
@@ -16,12 +16,10 @@
 use Psr\Log\NullLogger;
 
 /**
- * Factory provides method to create locks.
- *
  * @author Jérémy Derussé <jeremy@derusse.com>
  * @author Hamza Amrouche <hamza.simperfit@gmail.com>
  */
-class LockFactory implements LoggerAwareInterface
+class LockFactory implements LoggerAwareInterface, LockFactoryInterface
```

Comment:
If we have an interface for the factory, there should not be a reason to extend the class anymore. I suggest we add

```
@final since Symfony 5.3
```

Reply:
Thank you